apps/app/src/app/account/account-dbs/account-dbs.component.ts

```diff
@@ -12,43 +11,55 @@ import { TokenDialogComponent } from '../../components/token-dialog/token-dialog
 @Component({
   selector: 'app-account-dbs',
   templateUrl: './account-dbs.component.html',
-  styleUrls: ['./account-dbs.component.scss']
+  styleUrls: ['./account-dbs.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class AccountDBsComponent implements OnInit {
 
-  @Input() account: Account | undefined;
   @Input() zones: Zone[] | undefined;
 
-  nucliaDBs: NucliaDBMeta[] | undefined;
+  accountSlug = this.stateService.account.pipe(
+    filter((account) => !!account),
+    map((account) => account!.slug),
+    take(1)
+  );
+
+  nucliaDBs?: NucliaDBMeta[];
   maxNucliaDBs: number = 1;
   expandedDBs: string[] = [];
   unsubscribeAll = new Subject<void>();
 
-  constructor(private nucliaDBService: NucliaDBService, private dialog: MatDialog) { }
+  constructor(
+    private stateService: StateService,
+    private nucliaDBService: NucliaDBService,
+    private dialog: MatDialog,
+    private cdr: ChangeDetectorRef
+  ) { }
 
   ngOnInit(): void {
-    this.updateKeys();
+    this.updateKeys().subscribe();
```

Comment:
it would be safer to have a `takeUntil` here

Reply:
Fixed